runtime/tests/interpreter/capability_test.go

```diff
@@ -657,3 +657,46 @@ func TestInterpretCapability_check(t *testing.T) {
 		})
 	})
 }
+
+func TestInterpretCapability_address(t *testing.T) {
+
+	t.Parallel()
+
+	t.Run("check address", func(t *testing.T) {
+
+		t.Parallel()
+
+		inter, _ := testAccount(
+			t,
+			true,
+			`
+				fun single(): Address {
+					return account.getCapability(/public/single).address
+				}
+
+				fun double(): Address {
+					return account.getCapability(/public/double).address
+				}
+
+				fun nonExistent() : Address {
+					return account.getCapability(/public/nonExistent).address
+				}				
+			`,
+		)
+
+		t.Run("single", func(t *testing.T) {
+			_, err := inter.Invoke("single")
+			require.NoError(t, err)
```

Comment:
Here and in the tests below, assert that the result value, that is currently being ignored, is in fact of type `interpreter.AddressValue`:

```suggestion
			value, err := inter.Invoke("single")
			require.NoError(t, err)
			
			require.IsType(t, interpreter.AddressValue{}, value)
```

Reply:
Resolved in 11655c8